### DISCUSSION TITLE
Tranferring the data from company1.GitHub.com to company2.GitHub.com

### DISCUSSION THREAD

**Discussion by Venugopal1998** [community/community#52050] [New to GitHub]
Is there any possibility of switching/transferring of domain for below products?
Examples:
From company1.github.com to company2.github.com along with all resource such as repos, comments, branches, permissions etc...

**Comment by SMelidoni** [comment]
Hi there @Venugopal1998 👋🏽

GitHub doesn't provide a built-in way to directly transfer a domain from one organisation to another along with all the resources (repos, comments, branches, permissions, etc.) seamlessly. However, you can still achieve this in a few steps:

To transfer a domain and associated resources from one organisation to another, follow these steps:

1. **Transfer repositories**: Transfer all the repositories from `company1` to `company2` one by one. You can follow GitHub's [documentation on transferring a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) for guidance.
   Keep in mind that transferring a repository will retain the repository's issues, pull requests, and commit history. However, you'll need to update any links, remotes, and scripts that reference the old repository location.

2. **Update domain settings**: Update the custom domain settings in both organisations to point to the new domain.
   For `company1.github.com`, remove the domain settings from the `company1` organisation. For `company2.github.com`, configure the custom domain in the `company2` organisation. You can follow GitHub's [documentation on configuring a custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site) for guidance.

3. **Update DNS records**: Update your DNS provider's records to point the custom domain to GitHub Pages' servers. Follow GitHub's [documentation on setting up a custom domain with GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site) for guidance.

4. **Update permissions**: Manually update the permissions for the transferred repositories in the `company2` organisation to match the permissions in the `company1` organisation. You may need to invite the necessary collaborators, teams, or organisation members and assign them the appropriate roles.

Please note that this process may take some time and may require coordination between the administrators of both organisations. Make sure to communicate the changes to all relevant stakeholders to avoid confusion and potential downtime.

I hope this helps!
